bitnami/valkey-cluster/templates/update-cluster.yaml

```diff
@@ -141,16 +141,16 @@ spec:
                   newNodeIP=$(wait_for_dns_lookup "{{ template "common.names.fullname" . }}-${newNodeIndex}.{{ template "common.names.fullname" . }}-headless" 120 5)
                   sleep 5
                 done
-                slave=()
-                # when the index of the new node is less than `slaveNodesEndPos`，the added node is a replica that assigned to original valkey master node
-                # when the index of the new node is greater than or equal to `slaveNodesEndPos`，and it is not a multiple of `$VALKEY_CLUSTER_REPLICAS + 1`, the added node is a replica that assigned to newly added master node
-                if (( $VALKEY_CLUSTER_REPLICAS >= 1 )) && (( (( $newNodeIndex < $slaveNodesEndPos )) || (( (( $newNodeIndex >= $slaveNodesEndPos )) && (( $newNodeIndex % (( $VALKEY_CLUSTER_REPLICAS + 1 )) )) )) )); then
-                  slave+=("--cluster-slave")
+                replica=()
+                # when the index of the new node is less than `replicaNodesEndPos`，the added node is a replica that assigned to original valkey primary node
+                # when the index of the new node is greater than or equal to `replicaNodesEndPos`，and it is not a multiple of `$VALKEY_CLUSTER_REPLICAS + 1`, the added node is a replica that assigned to newly added primary node
+                if (( $VALKEY_CLUSTER_REPLICAS >= 1 )) && (( (( $newNodeIndex < $replicaNodesEndPos )) || (( (( $newNodeIndex >= $replicaNodesEndPos )) && (( $newNodeIndex % (( $VALKEY_CLUSTER_REPLICAS + 1 )) )) )) )); then
+                  replica+=("--cluster-slave")
```

Comment:
I guess here the "slave" term still applies, right?

Reply:
Yes, I have not found a replacement for that flag, and that's working, so I prefer to keep it.